lib/next-amp/README.md

```diff
@@ -0,0 +1,322 @@
+# AMP Next.js Library
```

Comment:
I am not checking all this - I assume identical to the separate repo you used to have.

Reply:
yup